tms/scalar/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""Scalar entrypoint."""
```

Comment:
General thoughts on putting everything in `__init__.py` vs a named submodule?  Stylistically it feels a bit weird to me.

Reply:
I was wondering what you would think of this. I've seen this pattern in a few source packages. It saves an extra file (since `__init__.py` is always present), but also does obfuscate important code. Using fewer files is sleeker, but is it worth the tradeoff? The other option is to move it to a `scalar/scalar.py` file.

Reply:
I like the better organization of `scalar/scalar.py`.  Mostly I use the `__init__.py` file for constants, or just leave it empty.  It's not a place I look for code.

Reading some perspectives on the internet, it seems it depends on people's backgrounds and what language they came from.  There is no official style rule about it.

Reply:
> people's backgrounds and what language they came from
That would explain it. I've added `scalar/scalar.py`